### PR TITLE
update_social_info

### PR DESCRIPTION
**Pull request by junpenglao**
Cancelled Travis builds as no code change.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -123,13 +123,13 @@ https://doi.org/10.7717/peerj-cs.55
 Contact
 =======
 
-To report an issue with PyMC3 or to suggest a feature please use the `issue tracker <https://github.com/pymc-devs/pymc3/issues>`__.
+We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel. You can also follow us on `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ for update and other announcements. Our `pymc Gitter channel <https://gitter.im/pymc-devs/pymc>`__ is still open if you want to have a quick interaction with PyMC3 developers and users.
```
Comment (fonnesbeck) [inline README.rst]: "for updates"

Comment (fonnesbeck) [reply README.rst]: Do we even want to advertise the Gitter channel anymore? I've stopped monitoring it.

File: README.rst [hunk 2]
```diff
@@ -123,13 +123,13 @@ https://doi.org/10.7717/peerj-cs.55
...
 
-To ask a question regarding modeling or usage of PyMC3 we encourage posting to `StackOverflow using the "pymc" tag <http://stackoverflow.com/questions/tagged/pymc>`__.
+To ask a question regarding modeling or usage of PyMC3 we encourage posting to Our Discourse forum under the `“Questions” Category <https://discourse.pymc.io/c/questions>`__. You can also suggest feature in the `“Development” Category <https://discourse.pymc.io/c/development>`__.
```
Comment (fonnesbeck) [inline README.rst]: "our" should be lower case

File: README.rst (hunk 1 above)
Comment (junpenglao) [reply README.rst]: I dont know about the Gitter - any long discussion we will direct them to discourse anyway. I guess it's still useful if ppl want to ask for quick advice?

**Comment by junpenglao** [comment]
@fonnesbeck I removed the Gitter channel information (asked a question on Gitter about this and no response for 2 days...).

**Comment by twiecki** [comment]
Looks great, thanks @junpenglao.

**Pull request merged**